controllers/operatorpolicy_status.go

```diff
@@ -856,19 +857,51 @@ func foundNotWantedObj(obj client.Object) policyv1.RelatedObject {
 	}
 }
 
-// foundNotWantedIpObj returns a Compliant RelatedObject with
+// foundNotApplicableObj returns a Compliant RelatedObject with
 // reason = 'Resource found but will not be handled in mustnothave mode'
 func foundNotApplicableObj(obj client.Object) policyv1.RelatedObject {
 	return policyv1.RelatedObject{
 		Object:    policyv1.ObjectResourceFromObj(obj),
 		Compliant: string(policyv1.Compliant),
-		Reason:    reasonIgnoreNotApplicable,
+		Reason:    reasonFoundNotApplicable,
 		Properties: &policyv1.ObjectProperties{
 			UID: string(obj.GetUID()),
 		},
 	}
 }
 
+// missingNotApplicableObj returns a conditionally Compliant RelatedObject with
+// reason = "Resource not found but should exist"/"Resource not found as expected"
+// based on the complianceType specified by the policy
+// Note: These relatedObjects are not applicable *only* in mustnothave mode
+func missingNotApplicableObj(name string, namespace string, complianceType policyv1.ComplianceType, gvk schema.GroupVersionKind) policyv1.RelatedObject {
```

Comment:
Can this be renamed to just `missingObj`? Since it doesn't use the NotApplicable reason, it seems odd.